src/setcovering.jl

```diff
@@ -60,7 +56,7 @@ The `cover` function performs a set covering algorithm on a collection of sub-co
 - `cx`: The complexity of the selected sub-covers.
 
 """
-function cover(sub_covers::AbstractVector{SubCover{N, T}}; max_itr::Int = 2, min_complexity::TF = 1.0) where{N, T, TF}
+function cover(sub_covers::AbstractVector{SubCover{N, T}}, max_itr::Int; min_complexity::TF = 1.0) where{N, T, TF}
```

Comment:
why `max_itr` is needed here?

Reply:
For similar reason above, the LP process have to be done iteratively.